packages/server/src/users/UsersModule.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
+
+import { FavouriteDishListController, FavouriteRestaurantListController } from './favourites/api';
+
+@Module({
+  controllers: [FavouriteRestaurantListController, FavouriteDishListController],
```

Comment:
Można by to wyrzucić do osobnego modułu jakby się za mocno rozrosło, czy może już teraz odseparować?

Reply:
Jak będzie przeszkadzać, to się porozdziela, ale wydaje mi się, że powinny być razem tak jak teraz.